src/user/User.js

```diff
@@ -182,7 +182,15 @@ const User = {
   _isTokenLogin: false,
   _isCallingLogin: false,
   _loginWithToken(value) {
-    Data._tokenIdSaved = value;
+    if (!value) {
+      Meteor.isVerbose &&
+        console.info(
+          'User._loginWithToken::: parameter value is null, do not save token.'
```

Comment:
Maybe do not say "do not save token" as it sounds imperative like "you shall not save token" :mage_man: which could be confusing

Maybe rather print "will not save as token"